english/monkey_hat/monkey_hats.md

```diff
@@ -0,0 +1,4 @@
+You see a monkey with a red hat staring at you...You are taken aback, but curious. The monkey appears to grin and beckon you. You
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.